src/hubbleds/story.py

```diff
@@ -427,5 +427,8 @@ def check_update(measurements):
 
     def setup_for_student(self, app_state):
         super().setup_for_student(app_state)
+        if self.student_user["id"] in range(2989, 3018):
+            print("Warning! You are using the ID of a beta test student.")
+        app_state.update_db = False
```

Comment:
I think you mean for line 432 to be indented under the if statement?

Reply:
Also, it looks like 2989 triggers the print warning, but 3018 does not, so the range is inclusive of 2989 but not 3018? I think that's what you intended, since 2989 is a test student and it looks like 3018 is not, but just wanted to make sure.

Reply:
Yes! That was a bug in line 432. As for the range, 3017 is the last beta student, so 3018 should be the second argument to `range` (which is inclusive of the start, but not the end).